pkgs/development/python-modules/pyairvisual/default.nix

```diff
@@ -38,8 +37,11 @@ buildPythonPackage rec {
   ];
 
   nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  checkInputs = [
```

Comment:
```suggestion
  nativeCheckInputs = [
```

Reply:
Time to ask the question that had been on my mind for a long time: should these *python* test deps in `nativeCheckInputs` or `checkInputs`? In some sense they are *linked* at test time, thus should be in `checkInputs`, just like why the runtime deps are in `buildInputs` (propagated), but the notion is just vague for python.

Reply:
cc @mweinelt @FRidh @jonringer

Reply:
Everything these days should go into `nativeBuildInputs`, at least all occurrences of `checkInputs` in our documentation were replaced like that.

Reply:
https://github.com/NixOS/nixpkgs/blob/633891ec46d41b4461945fa7a06b2a329313850a/doc/stdenv/stdenv.chapter.md?plain=1#L142-L144